.github/workflows/ci.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           libxrandr-dev
           libxtst-dev
           nasm
+      # This can be replaced with libsdl3-dev once the package is available
+      - name: Configure & Install SDL3
+        run: cd extern/SDL && cmake -S . -B build && cmake --build build --parallel "$(nproc)"
```

Comment:
we could build it in our cmake as described in [docs](https://github.com/libsdl-org/SDL/blob/main/docs/README-cmake.md#including-sdl-in-your-project) using the "vendored" variant:
```
add_subdirectory(path/to/SDL EXCLUDE_FROM_ALL)
```